docs/src/main/asciidoc/quartz.adoc

```diff
@@ -0,0 +1,331 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Scheduling Periodic Clustered Tasks
```

Comment:
It will be a guide about this only? Won't we expose more Quartz features?

Reply:
Should it be renamed to `Quarkus - Scheduling Periodic Tasks with Quartz` ?

Reply:
AFAIK we don't expose any Quarzt APIs at the moment and I think it's reasonable to keep it this way because we'd like to encourage users to continue using our scheduler extension API. Therefore it's ok to name the guide like this.

Reply:
@mkouba Yes that makes sense. I have already renamed it in the way I proposed above. I am also thinking of adding a link from the current scheduler extension to this guide with something like - use the quartz extension to schedule clustered jobs

WDYT?

Reply:
>  I have already renamed it in the way I proposed above...

Well, my comment was rather that it's ok to use "clustered" in the title because that's what this extension is for, ie. the message should be something like "if you really need clustered scheduler use the quartz extension".

> I am also thinking of adding a link from the current scheduler extension to this guide with something like...

I like it! Both guides should probably reference each other?

Reply:
Great. I’ll add it later on, thanks.

Reply:
I have added the links to both guides.

Reply:
OK, sorry, I thought the whole point of this extension was to expose the Quartz APIs at some point.